doc/sphinx/history.rst

```diff
@@ -1303,7 +1303,7 @@ Symbolic notations
 - "Print Scope sc" and "Locate ntn" allows to know to what expression a
   notation is bound
 - New defensive strategy for printing or not implicit arguments to ensure
-  re-type-checkability of the printed term
+  re-type checkability of the printed term
```

Comment:
This reads very strangly.  I think the hyphen should be left here, since otherwise it sounds like it's checking retyping rather than rechecking typing.